### DISCUSSION TITLE
Ao importar arquivo mockon disponibilizado só da erro

### DISCUSSION THREAD

**Discussion by ReneSaboiaMendesFilho** [balta-io/7187#60] [Perguntas e Respostas] [answered]
Baixei o aplicativo e tentei importar o arquivo disponibilizado e aparece que ele não é valido!

**Comment by andrebaltieri** [comment, marked as answer]
@ReneSaboiaMendesFilho poderia por favor postar um print ou mais detalhes do erro?

**Reply by ReneSaboiaMendesFilho** [reply]
Deu certo, ao invés de importar, abri o environment. Obrigado!

**Reply by andrebaltieri** [reply]
Perfeito <3

**Reply by ReneSaboiaMendesFilho** [reply]
Deu certo, mas o Body ficou assim:
![Mock](https://user-images.githubusercontent.com/85502566/145347150-b7c7040c-390d-4b3b-998e-413582e79deb.png)

**Comment by andrebaltieri** [comment]
@ReneSaboiaMendesFilho você importou qual JSON? Este aqui: https://github.com/balta-io/7187/blob/master/assets/7181.mockoon.1.4.0.json

**Reply by ReneSaboiaMendesFilho** [reply]
Sim este mesmo.
![image](https://user-images.githubusercontent.com/85502566/145679134-fa51e0a0-85a3-4e6a-ab55-e453d8dbc68f.png)
![image](https://user-images.githubusercontent.com/85502566/145679150-d34180ad-ded8-4621-beba-4d2ad5286506.png)

**Comment by BrewertonSantos** [comment]
@ReneSaboiaMendesFilho
Desculpe a demora em responder sua questão. Buscando resolver a dúvida, refiz o arquivo, exportei para JSON e testei a importação no Mockon assim como testei as requisições via PostMan. Segue abaixo:

```json
{
    "source": "mockoon:1.17.0",
    "data": [
        {
            "type": "environment",
            "item": {
                "uuid": "ebeeeee9-91f7-49d2-b866-8ccbdf0f7cdc",
                "lastMigration": 19,
                "name": "7181",
                "endpointPrefix": "v1",
                "latency": 0,
                "port": 3000,
                "routes": [
                    {
                        "uuid": "be7794d4-7a47-496c-bea7-c53b327cce04",
                        "documentation": "Retorna uma lista detalhada de todos produtos cadastrados",
                        "method": "get",
                        "endpoint": "products",
                        "responses": [
                            {
                                "uuid": "ced97c76-ba2d-4acd-bbe6-d941667d8d13",
                                "body": "[\n  {\n    \"_id\": \"481598b848f447b4b923ca2ad405aa09\",\n    \"title\": \"Cama M1 Urban\",\n    \"category\": \"accessories\",\n    \"description\": [\n      \"<ul><li><span><b>Pet:</b></span><span> Cachorros</span></li><li><span><b>Porte de Raça:</b></span><span> Grande, Médio e Pequeno</span></li><li><span><b>Idade:</b></span><span> Sênior, Adulto e Filhote</span></li><li><span><b>Material:</b></span><span> Plástico</span></li><li><span><b>Medidas<br>aproximadas:</b></span><span> Tamanho 8 (84 x 59,5 x h 25 cm), Tamanho 6 (70,5 x 52 x h 23,5 cm), Tamanho 4 (61,5 x 45 x h 21,5 cm), Tamanho 2 (52 x 36 x h 17,5 cm), Tamanho 12 (111 x 80,5 x h 33,5 cm), Tamanho 10 (93,5 x 68 x h 28 e5 cm)</span></li><li><span><b>Tipo:</b></span><span> Cama</span></li><li><span><b>Cor:</b></span><span> Bordô</span></li><li><span><b>Apresentação:</b></span><span> <p><span>Disponível nos tamanhos 2, 4, 6, 8, 10 e 12</span></p></span></li><li><span><b>Código de barras:</b></span><span> 8010690074689</span></li></ul>\\\",\\n        \\\"price\\\":5999,\\n        \\\"images\\\":[\\n            \\\"https://baltaio.blob.core.windows.net/courses/7180/cama-m1-urban-puppy.jpg\"\n    ]\n  },\n  {\n    \"_id\": \"481598b848f447b4b923ca2ad405aa10\",\n    \"title\": \"Ração Golden Special Sabor Frango e Carne para Cães Adultos\",\n    \"category\": \"food\",\n    \"description\": \"<p>- Indicado para cães adultos;<br>- Livre de corantes e aromatizantes artificiais;<br>- Alimento rico em nutrientes;<br>- Promove o equilíbrio intestinal;<br>- Protege as articulações;<br>- Disponível em embalagens de 15kg e 20kg.</p>\",\n    \"price\": 10490,\n    \"images\": [\n      \"https://baltaio.blob.core.windows.net/courses/7180/racao-golden.png\"\n    ]\n  },\n  {\n    \"_id\": \"481598b848f447b4b923ca2ad405aa11\",\n    \"title\": \"Ração Royal Canin Golden Retriever - Cães Adultos - 12kg\",\n    \"category\": \"food\",\n    \"description\": \"<p>- Indicada para cães adultos; <br>- Específica para pets da raça Golden Retriever; <br>- Auxilia na manutenção do peso ideal; <br>- Mantém a saúde da musculatura cardíaca; <br>- Proporciona uma pele e pelagem mais saudáveis,<br>- Disponível em embalagem de 12kg. </p>\",\n    \"price\": 21999,\n    \"images\": [\n      \"https://baltaio.blob.core.windows.net/courses/7180/racao-royal-canin.png\"\n    ]\n  }\n]",
                                "latency": 2000,
                                "statusCode": 200,
                                "label": "",
                                "headers": [],
                                "filePath": "",
                                "sendFileAsBody": false,
                                "rules": [],
                                "rulesOperator": "OR",
                                "disableTemplating": false,
                                "fallbackTo404": false
                            }
                        ],
                        "enabled": true,
                        "randomResponse": false,
                        "sequentialResponse": false
                    },
                    {
                        "uuid": "ddb5f82f-6ceb-4753-b59f-dca869e52018",
                        "documentation": "Retorna uma lista detalhada de todos pets cadastrados",
                        "method": "get",
                        "endpoint": "pets",
                        "responses": [
                            {
                                "uuid": "76d12bc7-eb41-42e4-81f5-368b11d2e331",
                                "body": "{}",
                                "latency": 800,
                                "statusCode": 200,
                                "label": "",
                                "headers": [],
                                "filePath": "",
                                "sendFileAsBody": false,
                                "rules": [],
                                "rulesOperator": "OR",
                                "disableTemplating": false,
                                "fallbackTo404": false
                            }
                        ],
                        "enabled": true,
                        "randomResponse": false,
                        "sequentialResponse": false
                    },
                    {
                        "uuid": "4bde5af7-4ecd-4d5a-b708-e8f3b4468950",
                        "documentation": "",
                        "method": "post",
                        "endpoint": "accounts/authenticate",
                        "responses": [
                            {
                                "uuid": "7aaf22de-b7d5-4cea-b63d-6765ef500fb2",
                                "body": "{\n  \"customer\": {\n    \"_id\": \"e064caf4b91e4d908a388a734a34feb3\",\n    \"name\": \"Bruce Wayne\",\n    \"document\": \"04052789008\",\n    \"email\": \"customer@petshop.com\"\n  },\n  \"token\": \"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ.eyJpZCI6IjVjODkwMzM4YjljMWYyNDI0Y2E2NDVmZCIsImVtYWlsIjoiYW5kcmViYWx0aWVyaUBnbWFpbC5jb20iLCJuYW1lIjoiQW5kcsOpIEJhbHRpZXJpIiwiZG9jdW1lbnQiOiIzNTI5MjYyMzgyMiIsInN1YnNjcmliZXIiOnRydWUsImlhdCI6MTU1NDMyMTg3MiwiZXhwIjoxNTU0NDA4MjcyfQ.YLS527i8yXnimL15jPyT8mkFAOYI7yorqh-zQyHq-N0\"\n}",
                                "latency": 2000,
                                "statusCode": 200,
                                "label": "",
                                "headers": [],
                                "filePath": "",
                                "sendFileAsBody": false,
                                "rules": [],
                                "rulesOperator": "OR",
                                "disableTemplating": false,
                                "fallbackTo404": false
                            }
                        ],
                        "enabled": true,
                        "randomResponse": false,
                        "sequentialResponse": false
                    },
                    {
                        "uuid": "72292285-9cb3-4618-b35f-7de559eadb84",
                        "documentation": "",
                        "method": "post",
                        "endpoint": "accounts/refresh-token",
                        "responses": [
                            {
                                "uuid": "76186f83-70f3-48e4-90f1-cf83fdb7ae93",
                                "body": "{\n  \"customer\": {\n  \"_id\": \"e064caf4b91e4d908a388a734a34feb3\",\n  \"name\": \"Bruce Wayne\",\n  \"document\": \"04052789008\",\n  \"email\": \"customer@petshop.com\"\n    \n  },\\\n  \"token\": \"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjVjODkwMzM4YjljMWYyNDI0Y2E2NDVmZCIsImVtYWlsIjoiYW5kcmViYWx0aWVyaUBnbWFpbC5jb20iLCJuYW1lIjoiQW5kcsOpIEJhbHRpZXJpIiwiZG9jdW1lbnQiOiIzNTI5MjYyMzgyMiIsInN1YnNjcmliZXIiOnRydWUsImlhdCI6MTU1NDMyMTg3MiwiZXhwIjoxNTU0NDA4MjcyfQ.YLS527i8yXnimL15jPyT8mkFAOYI7yorqh-zQyHq-N0\"\n  \n}",
                                "latency": 2000,
                                "statusCode": 200,
                                "label": "",
                                "headers": [],
                                "filePath": "",
                                "sendFileAsBody": false,
                                "rules": [],
                                "rulesOperator": "OR",
                                "disableTemplating": false,
                                "fallbackTo404": false
                            }
                        ],
                        "enabled": true,
                        "randomResponse": false,
                        "sequentialResponse": false
                    },
                    {
                        "uuid": "29c67810-0013-4e49-8c5f-6abdb1f1f765",
                        "documentation": "",
                        "method": "post",
                        "endpoint": "accounts",
                        "responses": [
                            {
                                "uuid": "3ec83d67-971e-4978-a253-e40877d97eac",
                                "body": "{\n  \"message\": \"Bem-vindo ao Dachshop!\"\n}",
                                "latency": 800,
                                "statusCode": 200,
                                "label": "",
                                "headers": [],
                                "filePath": "",
                                "sendFileAsBody": false,
                                "rules": [],
                                "rulesOperator": "OR",
                                "disableTemplating": false,
                                "fallbackTo404": false
                            }
                        ],
                        "enabled": true,
                        "randomResponse": false,
                        "sequentialResponse": false
                    },
                    {
                        "uuid": "71b2e54a-d641-40e3-a967-71a49c78196f",
                        "documentation": "",
                        "method": "post",
                        "endpoint": "accounts/reset-password",
                        "responses": [
                            {
                                "uuid": "1faf2cac-8a3c-41bb-9a6b-4d5c4a86efbf",
                                "body": "{\n  \"message\": \"Uma nova senha foi gerada e enviada para seu E-mail!\"\n}",
                                "latency": 1000,
                                "statusCode": 200,
                                "label": "",
                                "headers": [],
                                "filePath": "",
                                "sendFileAsBody": false,
                                "rules": [],
                                "rulesOperator": "OR",
                                "disableTemplating": false,
                                "fallbackTo404": false
                            }
                        ],
                        "enabled": true,
                        "randomResponse": false,
                        "sequentialResponse": false
                    },
                    {
                        "uuid": "77f66c88-1a93-456d-8b43-f7a07b99cf3c",
                        "documentation": "",
                        "method": "put",
                        "endpoint": "accounts",
                        "responses": [
                            {
                                "uuid": "ddcada67-3f39-4054-8fc3-a3a77e2f9ba9",
                                "body": "{\n  \"message\": \"Dados atualizados com sucesso!\"\n}",
                                "latency": 1000,
                                "statusCode": 200,
                                "label": "",
                                "headers": [],
                                "filePath": "",
                                "sendFileAsBody": false,
                                "rules": [],
                                "rulesOperator": "OR",
                                "disableTemplating": false,
                                "fallbackTo404": false
                            }
                        ],
                        "enabled": true,
                        "randomResponse": false,
                        "sequentialResponse": false
                    },
                    {
                        "uuid": "a93bd6e2-fe54-42f6-9c57-9e87d2a07bd5",
                        "documentation": "",
                        "method": "get",
                        "endpoint": "accounts",
                        "responses": [
                            {
                                "uuid": "bad08e59-d7aa-4516-877f-d1881c3639f7",
                                "body": "{\n  \"name\": \"Bruce Wayne\",\n  \"document\": \"015.418.850-66\",\n  \"email\": \"batman@balta.io\"\n}",
                                "latency": 1200,
                                "statusCode": 200,
                                "label": "",
                                "headers": [],
                                "filePath": "",
                                "sendFileAsBody": false,
                                "rules": [],
                                "rulesOperator": "OR",
                                "disableTemplating": false,
                                "fallbackTo404": false
                            }
                        ],
                        "enabled": true,
                        "randomResponse": false,
                        "sequentialResponse": false
                    }
                ],
                "proxyMode": false,
                "proxyHost": "",
                "proxyRemovePrefix": false,
                "tlsOptions": {
                    "enabled": false,
                    "type": "CERT",
                    "pfxPath": "",
                    "certPath": "",
                    "keyPath": "",
                    "caPath": "",
                    "passphrase": ""
                },
                "cors": true,
                "headers": [
                    {
                        "key": "Content-Type",
                        "value": "application/json"
                    },
                    {
                        "key": "Access-Control-Allow-Origin",
                        "value": "*"
                    },
                    {
                        "key": "Access-Control-Allow-Methods",
                        "value": "GET, POST, PUT, DELETE"
                    }
                ],
                "proxyReqHeaders": [],
                "proxyResHeaders": []
            }
        }
    ]
}
```

**Reply by ReneSaboiaMendesFilho** [reply]
Funcionou perfeitamente. Obrigado!

**Reply by andrebaltieri** [reply]
💜 Muito obrigado @BrewertonSantos !

**Comment by KarinaBurin** [comment]
Olá Bom dia, @andrebaltieri  por favor, me tire uma duvida, visto que tive o mesmo problema que o Rene, mesmo pegando o arquivo refeito pelo o @BrewertonSantos ainda sim não consigo importar o JSON. 

Segue prints

![image](https://user-images.githubusercontent.com/62036795/151697784-4a1a1f4c-a0f0-4022-b61b-10053ac8cdef.png)

o arquivo que estou importante eu substitui pelo o que o @BrewertonSantos refez, mas também não rolou.

![image](https://user-images.githubusercontent.com/62036795/151697798-a2929a81-a986-40ae-943b-7b0d3708516b.png)

O que estou fazendo de errado por favor?

![image](https://user-images.githubusercontent.com/62036795/151697831-36903f6c-2869-43ad-a4ca-eead154907a3.png)

**Reply by andrebaltieri** [reply]
@BrewertonSantos consegue ajudar nisso?

**Reply by KarinaBurin** [reply]
@andrebaltieri  Estou no aguardo... obrigada.

**Reply by andrebaltieri** [reply]
![image](https://user-images.githubusercontent.com/965305/152648431-39d88d8c-8c30-4359-9cb6-62dfe0f09693.png)
Boa tarde, @KarinaBurin, como vai?

Fiz o import aqui e deu certo... de qualquer forma, exportei um novo arquivo que pode ser encontrado aqui:
https://github.com/balta-io/7187/blob/master/assets/7187_2022-02-05.json

**Reply by GabrielWerlich** [reply]
![image](https://user-images.githubusercontent.com/40069738/158588577-ba5b05f9-8b61-4c43-8893-476f7fceff5d.png)


Tentei importar aqui também ambos os arquivos estão dando problema ao importar, tentei o que está na pasta antiga (arquivo 7181.mockoon.1.4.0) e também tentei o ultimo que o Balta disponibilizou no reply dia 05/fev mas também não deu certo

**Reply by andrebaltieri** [reply]
![import-mockoon](https://user-images.githubusercontent.com/965305/158599580-fe2c816a-8263-45bc-9e2d-57b1b0c12b7b.gif)
Fiz o teste aqui e funcionou corretamente!

**Reply by GabrielWerlich** [reply]
Perdão Balta eu estava importando pelo Import/export ao invés de criar um novo enviroment, enfim deu certo! Obrigado

**Comment by luisxfelipe** [comment]
Eu também tive o mesmo problema, a questão é que mesmo baixando o arquivo do git e importando não aparece nada na tela de produtos e eu até achava poder ser algo na atualização do Angular. Quando observei no Insomnia (equivalente ao Postman) vi que o primeiro produto estava quebrado, então eu consertei e ainda coloquei imagens direto site do Pets, já que o link do arquivo parece não existir mais.

Se alguém tiver o mesmo problema pegue esse arquivo do link abaixo, renomeie para .json (pois o git não deixou colocar .json) e importa no mockoon.

[mockoon-pets.txt](https://github.com/balta-io/7187/files/9388435/mockoon-pets.txt)

Ou tente copiar o código abaixo se não quebrar nada:

`{
  "uuid": "fa81cb6c-569a-4f35-9c4f-625be20581ac",
  "lastMigration": 22,
  "name": "7181",
  "endpointPrefix": "v1",
  "latency": 0,
  "port": 3000,
  "routes": [
    {
      "uuid": "6e47ae91-3c26-47a0-9403-ea7441ba22cf",
      "documentation": "Retorna uma lista detalhada de todos produtos cadastrados",
      "method": "get",
      "endpoint": "products",
      "responses": [
        {
          "uuid": "b14ff1e6-6828-4aac-a619-28ffe80bfccf",
          "body": "[\n  {\n    \"_id\": \"481598b848f447b4bsds54ad405aa10\",\n    \"title\": \"Cama M1 Urban Puppy para Cães Microfibra Lisa Marinho\",\n    \"category\": \"food\",\n    \"description\": \"<p>- Indicada para cães e gatos;<br>- Enchimento removível;<br>- Fibra siliconada;<br>- Estampa exclusiva;<br>- Super confortável e quentinha;<br>- Disponível nos tamanhos P, M e G.</p>\",\n    \"price\": 13490,\n    \"images\": [\n      \"https://static.petz.com.br/fotos/1525443192336.jpg\"\n    ]\n  },\n  {\n    \"_id\": \"481598b848f447b4b923ca2ad405aa10\",\n    \"title\": \"Ração Golden Special Sabor Frango e Carne para Cães Adultos\",\n    \"category\": \"food\",\n    \"description\": \"<p>- Indicado para cães adultos;<br>- Livre de corantes e aromatizantes artificiais;<br>- Alimento rico em nutrientes;<br>- Promove o equilíbrio intestinal;<br>- Protege as articulações;<br>- Disponível em embalagens de 15kg e 20kg.</p>\",\n    \"price\": 10490,\n    \"images\": [\n      \"https://static.petz.com.br/fotos/1658429012839.jpg\"\n    ]\n  },\n  {\n    \"_id\": \"481598b848f447b4b923ca2ad405aa11\",\n    \"title\": \"Ração Royal Canin Golden Retriever - Cães Adultos - 12kg\",\n    \"category\": \"food\",\n    \"description\": \"<p>- Indicada para cães adultos; <br>- Específica para pets da raça Golden Retriever; <br>- Auxilia na manutenção do peso ideal; <br>- Mantém a saúde da musculatura cardíaca; <br>- Proporciona uma pele e pelagem mais saudáveis,<br>- Disponível em embalagem de 12kg. </p>\",\n    \"price\": 21999,\n    \"images\": [\n      \"https://static.petz.com.br/fotos/1660162896325.jpg\"\n    ]\n  }\n]",
          "latency": 10,
          "statusCode": 200,
          "label": "",
          "headers": [],
          "filePath": "",
          "sendFileAsBody": false,
          "rules": [],
          "rulesOperator": "OR",
          "disableTemplating": false,
          "fallbackTo404": false,
          "default": true
        }
      ],
      "enabled": true,
      "responseMode": null
    },
    {
      "uuid": "30fbf2d9-6193-449c-abf4-8294760aaf8b",
      "documentation": "Retorna uma lista detalhada de todos pets cadastrados",
      "method": "get",
      "endpoint": "pets",
      "responses": [
        {
          "uuid": "9c183d5d-dc88-4558-bf52-489677ee1a61",
          "body": "{}",
          "latency": 800,
          "statusCode": 200,
          "label": "",
          "headers": [],
          "filePath": "",
          "sendFileAsBody": false,
          "rules": [],
          "rulesOperator": "OR",
          "disableTemplating": false,
          "fallbackTo404": false,
          "default": true
        }
      ],
      "enabled": true,
      "responseMode": null
    },
    {
      "uuid": "2225dc7f-c98f-4422-b1c5-2e7c3846b2ea",
      "documentation": "",
      "method": "post",
      "endpoint": "accounts/authenticate",
      "responses": [
        {
          "uuid": "6da21517-1999-49a2-aaca-0eb973b1ae79",
          "body": "{\n  \"customer\": {\n    \"_id\": \"e064caf4b91e4d908a388a734a34feb3\",\n    \"name\": \"Bruce Wayne\",\n    \"document\": \"04052789008\",\n    \"email\": \"customer@petshop.com\"\n  },\n  \"token\": \"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ.eyJpZCI6IjVjODkwMzM4YjljMWYyNDI0Y2E2NDVmZCIsImVtYWlsIjoiYW5kcmViYWx0aWVyaUBnbWFpbC5jb20iLCJuYW1lIjoiQW5kcsOpIEJhbHRpZXJpIiwiZG9jdW1lbnQiOiIzNTI5MjYyMzgyMiIsInN1YnNjcmliZXIiOnRydWUsImlhdCI6MTU1NDMyMTg3MiwiZXhwIjoxNTU0NDA4MjcyfQ.YLS527i8yXnimL15jPyT8mkFAOYI7yorqh-zQyHq-N0\"\n}",
          "latency": 2000,
          "statusCode": 200,
          "label": "",
          "headers": [],
          "filePath": "",
          "sendFileAsBody": false,
          "rules": [],
          "rulesOperator": "OR",
          "disableTemplating": false,
          "fallbackTo404": false,
          "default": true
        }
      ],
      "enabled": true,
      "responseMode": null
    },
    {
      "uuid": "84fd78d6-247f-4329-859e-f2da2bf24fdd",
      "documentation": "",
      "method": "post",
      "endpoint": "accounts/refresh-token",
      "responses": [
        {
          "uuid": "be98bb93-9ce4-4f14-95f0-bbb1797e72a5",
          "body": "{\n  \"customer\": {\n  \"_id\": \"e064caf4b91e4d908a388a734a34feb3\",\n  \"name\": \"Bruce Wayne\",\n  \"document\": \"04052789008\",\n  \"email\": \"customer@petshop.com\"\n    \n  },\\\n  \"token\": \"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjVjODkwMzM4YjljMWYyNDI0Y2E2NDVmZCIsImVtYWlsIjoiYW5kcmViYWx0aWVyaUBnbWFpbC5jb20iLCJuYW1lIjoiQW5kcsOpIEJhbHRpZXJpIiwiZG9jdW1lbnQiOiIzNTI5MjYyMzgyMiIsInN1YnNjcmliZXIiOnRydWUsImlhdCI6MTU1NDMyMTg3MiwiZXhwIjoxNTU0NDA4MjcyfQ.YLS527i8yXnimL15jPyT8mkFAOYI7yorqh-zQyHq-N0\"\n  \n}",
          "latency": 2000,
          "statusCode": 200,
          "label": "",
          "headers": [],
          "filePath": "",
          "sendFileAsBody": false,
          "rules": [],
          "rulesOperator": "OR",
          "disableTemplating": false,
          "fallbackTo404": false,
          "default": true
        }
      ],
      "enabled": true,
      "responseMode": null
    },
    {
      "uuid": "ab9c004b-847b-42e4-861e-08f0763b4286",
      "documentation": "",
      "method": "post",
      "endpoint": "accounts",
      "responses": [
        {
          "uuid": "f615e0e7-0e76-4daf-a946-7c2a6a8832b9",
          "body": "{\n  \"message\": \"Bem-vindo ao Dachshop!\"\n}",
          "latency": 800,
          "statusCode": 200,
          "label": "",
          "headers": [],
          "filePath": "",
          "sendFileAsBody": false,
          "rules": [],
          "rulesOperator": "OR",
          "disableTemplating": false,
          "fallbackTo404": false,
          "default": true
        }
      ],
      "enabled": true,
      "responseMode": null
    },
    {
      "uuid": "0a5211ad-92b3-48d8-a70d-c9dde28862e2",
      "documentation": "",
      "method": "post",
      "endpoint": "accounts/reset-password",
      "responses": [
        {
          "uuid": "56870971-12a0-481f-b0d2-93add84fbf87",
          "body": "{\n  \"message\": \"Uma nova senha foi gerada e enviada para seu E-mail!\"\n}",
          "latency": 1000,
          "statusCode": 200,
          "label": "",
          "headers": [],
          "filePath": "",
          "sendFileAsBody": false,
          "rules": [],
          "rulesOperator": "OR",
          "disableTemplating": false,
          "fallbackTo404": false,
          "default": true
        }
      ],
      "enabled": true,
      "responseMode": null
    },
    {
      "uuid": "68c98d60-d415-4833-af8f-154619b2a4da",
      "documentation": "",
      "method": "put",
      "endpoint": "accounts",
      "responses": [
        {
          "uuid": "e727eff6-2c9c-4e0e-b2ad-5aedeb99036f",
          "body": "{\n  \"message\": \"Dados atualizados com sucesso!\"\n}",
          "latency": 1000,
          "statusCode": 200,
          "label": "",
          "headers": [],
          "filePath": "",
          "sendFileAsBody": false,
          "rules": [],
          "rulesOperator": "OR",
          "disableTemplating": false,
          "fallbackTo404": false,
          "default": true
        }
      ],
      "enabled": true,
      "responseMode": null
    },
    {
      "uuid": "c8c42fc4-41a6-44c8-a541-2959b7ee397b",
      "documentation": "",
      "method": "get",
      "endpoint": "accounts",
      "responses": [
        {
          "uuid": "afea87e4-848e-4862-ba77-476a530a8833",
          "body": "{\n  \"name\": \"Bruce Wayne\",\n  \"document\": \"015.418.850-66\",\n  \"email\": \"batman@balta.io\"\n}",
          "latency": 1200,
          "statusCode": 200,
          "label": "",
          "headers": [],
          "filePath": "",
          "sendFileAsBody": false,
          "rules": [],
          "rulesOperator": "OR",
          "disableTemplating": false,
          "fallbackTo404": false,
          "default": true
        }
      ],
      "enabled": true,
      "responseMode": null
    }
  ],
  "proxyMode": false,
  "proxyHost": "",
  "proxyRemovePrefix": false,
  "tlsOptions": {
    "enabled": false,
    "type": "CERT",
    "pfxPath": "",
    "certPath": "",
    "keyPath": "",
    "caPath": "",
    "passphrase": ""
  },
  "cors": true,
  "headers": [
    {
      "key": "Content-Type",
      "value": "application/json"
    },
    {
      "key": "Access-Control-Allow-Origin",
      "value": "*"
    },
    {
      "key": "Access-Control-Allow-Methods",
      "value": "GET, POST, PUT, DELETE"
    }
  ],
  "proxyReqHeaders": [],
  "proxyResHeaders": [],
  "hostname": "0.0.0.0"
}`

**Reply by andrebaltieri** [reply]
@luisxfelipe desta forma não deu certo 👇

![](https://user-images.githubusercontent.com/965305/158599580-fe2c816a-8263-45bc-9e2d-57b1b0c12b7b.gif)

**Reply by luisxfelipe** [reply]
O problema não está na forma como importa o arquivo e na requisição, pois no arquivo que foi disponibilizado no git na linha 19 está com algo de errado,  mais especificamente na descrição do 1produto "Cama M1"  e isso faz com que não apareça nem o preço nem imagens no json. 
Segue a foto de como fica importando o arquivo disponibilizado:

![Captura de tela 2022-08-23 162307](https://user-images.githubusercontent.com/31926895/186248267-c24fd24a-8112-4005-8203-a19744a28ece.png)

Segue a foto de como fica a requisição depois que arrumei:
![Captura de tela 2022-08-23 162734](https://user-images.githubusercontent.com/31926895/186249039-285d3e7b-9cb0-4cf5-ae4f-5d2b6fe1d306.png)

**Reply by andrebaltieri** [reply]
@luisxfelipe muito obrigado pelo feedback!! Você consegue exportar este arquivo corrigido e me passar?
